src/store/page/page-element.ts

```diff
@@ -465,13 +512,16 @@ export class PageElement {
 			pattern: this.patternId
 		};
 
-		json.children = this.children.map(
-			(element: PageElement) =>
-				// tslint:disable-next-line:no-any
-				element.toJsonObject ? element.toJsonObject() : (element as any)
-		);
-		json.properties = {};
+		json.contents = {};
+		this.contents.forEach((slotContents, slotId) => {
```

Comment:
I'd like to be picky about the words 'contents' and 'content' here. We should be consistent on how we use them.
We have 1. the Map<string, PageElement[]>, 2. one slot's PageElement[], 3. one slot's array item.
We should have a consistent wording for all three, and check the usage of "content[s]" in the source against it.

Reply:
done. `contents` = contents of all slots. `slotContents` = contents of a specific slot